README.md

```diff
@@ -22,25 +22,53 @@ After installing, you need to require Composer's autoloader:
 require 'vendor/autoload.php';
 ```
 
+## Setting up a Request Adapter
+Because of dependency collision, we have opted to use a request adapter rather than
+requiring a request library.  This means that your application will need to pass in
+a request adapter to the constructor of the SparkPost Library.  We use the [Ivory HTTP Adapter] (https://github.com/egeloen/ivory-http-adapter) in SparkPost. Please visit their repo
+for a list of supported adapters.  If you don't currently use a request library, you will
+need to require one and create an adapter from it and pass it along.  The example below uses the
+GuzzleHttp Client Library.
+
+An Adapter can be setup like so:
+```
+use SparkPost\SparkPost;
+use GuzzleHttp\Client;
+use Ivory\HttpAdapter\Guzzle6HttpAdapter;
+
+$httpAdapter = new Guzzle6HttpAdapter(new Client());
+$sparky = new SparkPost($httpAdapter, ['key'=>'YOUR API KEY']);
+```
+
 ## Getting Started:  Your First Mailing
 ```
-SparkPost::setConfig(["key"=>"YOUR API KEY"]);
+use SparkPost\SparkPost;
```

Comment:
would be nice to keep the `require 'vendor/autoload.php';` at the top here just for completeness...